lib/opus_support.py

```diff
@@ -1947,7 +1947,25 @@ def parse_form_type(s):
 
     return form_type, form_type_format, form_type_unit
 
-
+def get_single_parse_function(form_type_unit_id):

```

Comment:
Add docstring: """Return the parse func for a unit_id with a single non-displayed unit."""